### ISSUE TITLE
attempting to sync unrecognized object v1.Secret - error while cloning a repository containing `devfile.yaml` from `gitlab.org.com`

### ISSUE THREAD

**Issue by Divine1** [eclipse/che#21083]
### Summary

![image](https://user-images.githubusercontent.com/10482177/151430015-f4031473-8a4c-40fd-9572-08580f76af02.png)

i have installed above `eclipse-che` version in minikube cluster and installation is successful

i want to integrate `gitlab-oauth` with `eclipse-che`

l have applied below `gitlab-secret` in `eclipse-che` namespace.

```
kind: Secret
apiVersion: v1
metadata:
  name: gitlab-oauth-config
  namespace: 'eclipse-che' 
  labels:
    app.kubernetes.io/part-of: che.eclipse.org
    app.kubernetes.io/component: oauth-scm-configuration
  annotations:
    che.eclipse.org/oauth-scm-server: gitlab
    che.eclipse.org/scm-server-endpoint: 'https://gitlab.org.com'
type: Opaque
data:
  id: 'ZWFkNDhjNzE1ODQ1ZDU3NDI1M2ExOGRmZTVlOTlkMjhhYjViMmY2OGQ4NDY5ODZmMTQid'
  secret: 'OGM0MDMwNWQ2OWI1ZGQ1ZGQ3ODk0YWQ0ZDc1MjZmMzNjMmZiMTUwMzhiMDJkMGZsecret'
```
then i opened the `eclipse-che` website `https://192.168.49.2.nip.io`, then logged-in using `che@eclipse.org` / `admin`

After login, i provided my repository containing valid `devfile.yaml` `https://gitlab.org.com/demoproject.git` -> clicked create button -> this redirected the website from `eclipse-che` to `gitlab.org.com` , the `sso authentication` was successful. But the dev file cloning process failed due to this error `attempting to sync unrecognized object v1.Secret`

i have attached the screenshot showing error below. Please let me know how to fix this error.

![image](https://user-images.githubusercontent.com/10482177/151431570-ed56545c-8f6c-4fc5-b6c4-283e5e2c08ab.png)

![image](https://user-images.githubusercontent.com/10482177/151400846-78a86a79-bcf6-4b44-9418-948db31a9e0d.png)

**Comment by benoitf** [comment]
related to https://github.com/devfile/devworkspace-operator/issues/751

**Comment by Divine1** [comment]
> related to [devfile/devworkspace-operator#751](https://github.com/devfile/devworkspace-operator/issues/751)
> 
> requires 0.12.0 operator or next channel

thank you for the response.

how do i use `0.12.0 operator` or `next` channel to fix this issue ?

**Comment by tolusha** [comment]
We are going to update che-operator to use DW 0.12.1 as a dependency
https://github.com/eclipse-che/che-operator/pull/1302

cc @amisevsk

**Comment by Divine1** [comment]
@tolusha @amisevsk 

 thank you for this update.
when is it expected to be released? 
will this be released as a new patch version? 
how do i install the new release? `chectl update next` ?

currently below is the version i'm using
![image](https://user-images.githubusercontent.com/10482177/151568779-f02a66c1-a1a9-4bd3-adeb-fa7f9781dc23.png)

**Comment by amisevsk** [comment]
The changes will land in Che `next` as soon as https://github.com/eclipse-che/che-operator/pull/1302 is merged. I'm not 100% on the process of that landing in chectl, @benoitf can confirm.

**Comment by benoitf** [comment]
7.43.0 stable release will happen next week
Chectl/next will also include these changes on the next build of chectl after operator pr will be merged

**Comment by Divine1** [comment]
@benoitf 

is the fix [mentioned here](https://github.com/eclipse/che/issues/21083#issuecomment-1023935494) released in this chectl version `0.0.20220128-next.0a7174a` ? or it is yet to be released?

![image](https://user-images.githubusercontent.com/10482177/151695571-5f3388a1-ad3b-433b-8622-8bf9682500b2.png)


i installed the chectl version `0.0.20220128-next.0a7174a` and still with `gitlab-org` sso issue exists..

![image](https://user-images.githubusercontent.com/10482177/151695642-d1fe8160-8656-4d41-844d-09495d9acdcd.png)

**Comment by Divine1** [comment]
@amisevsk  thank you for the response. i'm able to update eclipse-che server to the latest version.

i'm facing some issues, i will create a new issue.
Have a great day
